### PR TITLE
fix(types): exposed types from overlay

### PR DESCRIPTION
**Pull request by Shubhdeep12**
Fixes: #219 
Before opening this PR:
* [x] I added a [Changeset Entry](https://spotlightjs.com/contribute/changesets/) with `pnpm changeset:add`
* [x] I referenced issues that this PR addresses

### CODE REVIEW

File: packages/overlay/package.json
Comment (Lms24) [inline packages/overlay/package.json:1]: let's also set the top level `types` property in this file

File: .changeset/gold-numbers-remain.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@spotlightjs/spotlight': patch
+'@spotlightjs/overlay': patch
+---
+
+exports types from overlay
```
Comment (Lms24) [inline .changeset/gold-numbers-remain.md:6]:
```suggestion
fix: Exports types from overlay
```

**Review by Lms24 — COMMENTED** [review]
Thanks for fixing! Two small comments otherwise ready to go!

File: packages/overlay/package.json
Comment (Shubhdeep12) [reply packages/overlay/package.json:1]: Done @Lms24. Please check.

**Pull request merged**